BuyTests/Graph.Task.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  Graph.Task.swift
```

Comment:
`Task` is a nested class under the `Graph` namespace. It neatly groups all classes pertaining to GraphQL under one namespace.